octobot_trading/personal_data/trades/trades_manager.py

```diff
@@ -92,6 +96,17 @@ def _reset_trades(self):
         self.trades_initialized = False
         self.trades = collections.OrderedDict()
 
+    async def _load_trades_history(self):
+        if self.trader.exchange_manager.is_backtesting:
+            # don't load history on backtesting
+            return
+        try:
+            if self.trader.exchange_manager.storage_manager.trades_storage:
```

Comment:
Can `exchange_manager.storage_manager` be None ?

Reply:
Nope, it's initialized in exchange manager constructor

Reply:
Even if storage is disabled?

Reply:
when storage is disabled `self.trader.exchange_manager.storage_manager.trades_storage` is None and  `self.trader.exchange_manager.storage_manager` exists. But maybe you mean something else ?
![image](https://user-images.githubusercontent.com/9078616/218255634-14ca7735-ccd1-4fb3-9be2-7ed7cb85ee11.png)

Reply:
No it's ok thanks :+1: